### ISSUE TITLE
Accessibility: should add role=img to placeholder div if using.

### ISSUE THREAD

**Issue by alice** [Polymer/core-image#12]
If an img tag isn't used, the placeholder div should get an image role.

**Comment by ssorallen** [comment]
@alice, should the `role="img"` be applied to the host element (`<core-image>` itself) or to the `<div id="placeholder">` in the shadow DOM?

**Comment by alice** [comment]
It doesn't matter too much, really. I suggested putting it on the placeholder div because `<img>` obviously already has image semantics, so you might as well be consistent across both cases.

**Comment by ssorallen** [comment]
Do screen readers read from the shadow DOM currently? To be correct and work today, it seems like `role="img"` should be added and reflected on the host element.

I'm new to how ARIA attributes work with shadow DOM, so I'm hoping to understand the best approach here.

**Comment by alice** [comment]
Yes, screen readers and other assistive technology (other than ChromeVox) do read from Shadow DOM - they access the composed tree.

**Comment by ssorallen** [comment]
Okay, thanks for the clarification.
